src/nyc_trees/apps/core/templates/base.html

```diff
@@ -18,6 +18,11 @@
                     </div>
                 </div>
                 <ul>
+                    {% if my_events_now|length == 1 %}
+                        {% with event=my_events_now.0 %}
+                            <li><a href="{{ event.get_user_checkin_url }}">Start Mapping</a></li>
```

Comment:
As intended ( and described in a possibly-hard-to-find commit comment):
In the unlikely event of more than 1 "current" event we show nothing in the hamburger menu. It would be too verbose to spell out the event names, and there are 2 other ways to get to the "Start Mapping" page.